Workshop_Cheatsheets/3-TEST-WRITING.md

```diff
@@ -2,31 +2,58 @@
 
 Many great examples can be found [here](https://github.com/wix/Detox/tree/master/detox/test/e2e).
 
-## Matchers
+## Documentation
 
-[See documentation here.](https://wix.github.io/Detox/docs/api/matchers)
+- [Matchers](https://wix.github.io/Detox/docs/api/matchers)
+- [Actions](https://wix.github.io/Detox/docs/api/actions-on-element)
+- [Expectations](https://wix.github.io/Detox/docs/api/expect)
 
-## Adding testID
+## Best practices
 
-It is always the best idea to match your element by something unique. We recommend using testID prop for this purpose if possible. 
+- Add testID
 
-Note: not all React Native components support testID prop. 
+It is always the best idea to match your element by something unique. We recommend using testID prop for this purpose if possible. 
 
 ```jsx
 <View>
   <TouchableOpacity testID='MyUniqueId123'>
-    <Text>Some button</Text>
+    <Text>Some text</Text>
   </TouchableOpacity>
 </View>
 ```
 
-## Actions
+- Pass testID to native components
 
-[See documentation here.](https://wix.github.io/Detox/docs/api/actions-on-element)
+Not all React Native components support testID prop, so you'll need to pass them to your native components. 
```

Comment:
Custom React Native components are not aware of `testID`s you pass to them. You need to pass the testID props down and down until it reaches a native component (e.g. `<View>`) which implements `testID` rendering as native accessibility identifiers in iOS and Android.